apps/wildfires/src/app/pages/about/index.tsx

```diff
@@ -0,0 +1,98 @@
+import BaLogo from '../../../assets/images/ba-logo-white.png';
+import Imagine from '../../../assets/images/imagine_la_logo.png';
+import Mayor from '../../../assets/images/mayors-fund-logo.png';
+import ContactUs from '../../shared/components/contactUs/ContactUs';
+
+export default function About() {
+  return (
+    <div className="-mx-10">
+      <div
+        style={{
+          background:
+            'linear-gradient(90deg, rgba(30,51,66,1) 70%, rgba(156,220,237,1) 70%)',
+        }}
+      >
+        <div className="max-w-7xl mx-auto flex flex-col md:flex-row items-stretch">
+          <div className="bg-brand-dark-blue w-full md:w-[60%] py-40 flex items-center px-10 md:py-0">
+            <h1 className="border-l-[10px] border-brand-yellow font-extralight md:font-bold pl-8 text-5xl md:text-[74px] text-white leading-[96.2px] text-center">
+              About Wildfire LA
```

Comment:
header breaks onto two lines in mobile view

<img width="398" alt="image" src="https://github.com/user-attachments/assets/4903a5ee-da7b-4fbf-9940-c7f3d7a2316a" />